tianlutech/mediterranea-marine-nextjs
# Upload multiple pictures and convert to images

## Context


I added the functionality of uploading multiple pictures and then convert them into pdf and be attached on the email.


## Test



- Fill up the bill form as normal
- While you are uploading the images add multiple images 
- When submitting it should first convert them to pdfs
- Add then check your email there should be all the images converted to images



## Platform



## Proof of Concept
